scm/src/main/java/com/sandboni/core/scm/scope/analysis/VersionScanner.java

```diff
@@ -15,7 +15,7 @@ public boolean scan(Change change) {
             return false;
         }
 
-        String[] rows = change.getFileContent().orElse("").split(System.lineSeparator());
+        String[] rows = change.getFileContent().orElse("").split("\\R"); //covers Windows/Unix/Mac OS
```

Comment:
If the fileContent is empty, we will still have IndexOutOfBoundsException, right? better to check that scenario too.

Reply:
Not really. we check change.getFileContent().isPresent() a few lines before (and return false)